ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/list.md

```diff
@@ -595,3 +595,21 @@ FROM my_table;
 
 {% endif %}
 
+## ListTop, ListTopAsc, ListTopDesc, ListTopSort, ListTopSortAsc и ListTopSortDesc {#listtop}
+
+Select top values from the list. `ListTopSort*` additionally sorts the returned values. The smallest values are selected by default. Thus, the functions without a suffix are the aliases to `*Asc` functions, while `*Desc` functions return the largest values.
+
+ListTopSort is more effective that consecutive ListTop and ListSort (because ListTop can partially sort the list in order to find needed values), however ListTop is more effective than ListTopSort when the order of the result is not important.
```

Comment:
```suggestion
`ListTopSort` is more effective than consecutive `ListTop` and `ListSort` because `ListTop` can partially sort the list to find needed values. However, `ListTop` is more effective than `ListTopSort` when the result order is unimportant.
```